packages/sources/css/src/components/snackbar/src/index.css

```diff
@@ -0,0 +1,42 @@
+@import '@vtmn/css-design-tokens/src/base-colors';
+@import '@vtmn/css-design-tokens/src/themes/default';
+@import '@vtmn/css-design-tokens/src/spacings';
+@import '@vtmn/css-design-tokens/src/typography';
+@import '@vtmn/css-design-tokens/src/opacities';
+@import '@vtmn/css-design-tokens/src/shadows';
+@import '@vtmn/css-design-tokens/src/animations';
+
+.vtmn-snackbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: rem(8px) rem(16px);
+  height: rem(48px);
+  max-width: fit-content;
+  font-family: var(--vtmn-typo_font-family);
+  font-size: var(--vtmn-typo_text-2-font-size);
+  line-height: var(--vtmn-typo_text-2-line-height);
+  color: var(--vtmn-semantic-color_content-primary-reversed);
+  background-color: var(--vtmn-semantic-color_background-primary-reversed);
+  box-shadow: var(--vtmn-shadow_200);
+  border-radius: rem(4px);
+}
+
+.vtmn-snackbar.show {
+  visibility: visible;
+  position: fixed;
+  right: 50%;
+  bottom: 0;
+  transform: translate(50%, -50%);
+  animation: var(--vtmn-animation_overlay);
+}
+
+.vtmn-snackbar > .vtmn-btn {
+  margin-left: rem(16px);
+}
+
+.vtmn-snackbar_content {
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
```

Comment:
dont you have a dedicated tailwind helper for that ? 
vtmn-text-truncate or something ? https://tailwindcss.com/docs/text-overflow#truncate